dvc/command/completion.py

```diff
@@ -95,7 +96,7 @@ def run(self):
         parser = get_main_parser()
         shell = self.args.shell
         script = shtab.complete(parser, shell=shell, preamble=PREAMBLE)
-        print(script)
+        ui.write(script, force=True)
```

Comment:
Note that `force=True` will print completion script even in the `--quiet` mode (which what was already there with `print()`).